src/commands/tournament.ts

```
@@ -0,0 +1,591 @@
+}
+
+async function handleCreateTournament(
+  interaction: ChatInputCommandInteraction,
+  guildId: string
+): Promise<void> {
+  if (!(await ensureEventManager(interaction, guildId))) return;
+
+  const name = interaction.options.getString('name', true);
+  const playersPerMatch =
+    interaction.options.getInteger('players_per_match') ??
+    (config.tournaments?.defaultPlayersPerMatch ?? 3);
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)